stackslib/src/chainstate/stacks/index/test/marf.rs

```diff
@@ -77,8 +77,8 @@ fn marf_insert_different_leaf_same_block_100() {
         merkle_test_marf(
             &mut marf.borrow_storage_backend(),
             &block_header,
-            &path_bytes.to_vec(),
-            &[99; 40].to_vec(),
+            path_bytes.as_ref(),
+            [99; 40].as_ref(),
```

Comment:
Can prefix with `&` instead which IMO is cleaner:

```suggestion
            &path_bytes,
            &[99; 40],
```

This applies to every change in this file